commons/c2cgeoportal_commons/scripts/initializedb.py

```diff
@@ -30,8 +33,9 @@ def main(argv=sys.argv):
         usage(argv)
     config_uri = argv[1]
     options = parse_vars(argv[2:])
-    setup_logging(config_uri)
-    settings = get_appsettings(config_uri, options=options)
+
+    fileConfig(config_uri, defaults=os.environ)
+    settings = get_appsettings(config_uri, options=os.environ)
```

Comment:
This seems strange to me, some lines below we tests that there is a --force in options.
Maybe link with environment variables could be done in caller.